plugin/session_view.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, listener: AbstractViewListener, session: Session, uri: Docume
         if session_buffer is None:
             session_buffer = SessionBuffer(self, buffer_id, uri)
             self._session_buffers[key] = session_buffer
+            self._session.register_session_buffer_async(session_buffer)
```

Comment:
Previously `SessionBuffer` registered itself in `Session` with:

```py
self._session.register_session_buffer_async(self)
```

This caused crash as `Session.register_session_buffer_async` was called before `self._session_buffers[key] = session_buffer` and would crash depending what we would want to do with `SessionBuffer`.

I think it makes more sense for `SessionView` to register `SessionBuffer` with `Session` as it is also responsible for creating SB.